website/content/docs/commands/job/tag.mdx

```diff
@@ -0,0 +1,89 @@
+---
+layout: docs
+page_title: 'Commands: job tag'
+description: |
+  Use the job tag commands to manage versions. The job tag apply command saves a job version tag, and the job tag unset command removes a job version tag.
+---
+
+
+# Command: job tag
+
+@include 'version/tag-reason.mdx'
+
+Use the `job tag` command to manage tags for job versions. `job tag` has the
+following subcommands:
+
+- [`job tag apply`](#apply): Save a job version tag.
+- [`job tag unset`](#unset): Remove a tag from a job version.
+
+## Usage
+
+```shell
+nomad job tag <subcommand> [options] [args]
+```
+
+## General options
+
+Use these optional general options with the subcommands.
+
+@include 'general_options.mdx'
+
+## Apply
+
+Use `job tag apply` to create or modify a version tag.
+
+### Apply usage
+
+```shell-session
+nomad job tag apply [options] <job_id>
+```
+
+### Apply options
+
+- `name`: Specifies the name of the tag. Must be unique per job.
+- `description`: If set, specifies a description for the tag.
+- `version`: If set, specifies the version of the job to tag. If not provided, Nomad tags the latest version of the job.
+
+### Apply examples
+
+This example tags the latest version of the job `hello-world`.
+
+```shell-session
+$ nomad job tag apply -name "golden-version" \
+		-description "The version we can roll back to." \
+    hello-world
+```
+
+This example tags version zero of the job `hello-world`.
+
+```shell-session
+$ nomad job tag apply -version 0 \
+      -name "golden-version" \
+      hello-world
+```
+
+## Unset
+
+Use `nomad job tag unset` to delete a tag from a version. This command requires a job name and a tag name.
```

Comment:
That wasn't the intention of "unset"; I wanted to use "apply" and "unset" instead of "add" and "delete" because I worried users would confuse those verbs being about the version, rather than about the tag (and "applying" and "unsetting" a version doesn't make sense the way "adding" and "deleting" a version might)